packages/internals/src/__tests__/engine-commands/getConfig.test.ts

```diff
@@ -1,8 +1,19 @@
-import stripAnsi from 'strip-ansi'
-
 import { getConfig } from '../..'
+import { isRustPanic } from '../../panic'
 
 describe('getConfig', () => {
+  test('should raise a Rust panic when given arguments of the wrong type', async () => {
+    expect.assertions(1)
+
+    try {
+      // @ts-ignore
```

Comment:
Nit: I think `// @ts-expect-error` is usually better

Reply:
Fair enough, done!